content/en/docs/reference/glossary/external-control-plane.md

```diff
@@ -0,0 +1,10 @@
+---
+title: External Control Plane
+test: n/a
+---
+
+An external control plane is a [control plane](/docs/reference/glossary/#control-plane)
+that externally manages mesh workloads running in their own [clusters](/docs/reference/glossary/#cluster)
+or other infrastructure. The control plane may, itself, be deployed in a cluster, although not
+in one of the clusters that is part of the mesh it's controlling.
+Its purpose is to cleanly separate the control plane from the data plane of a mesh.
```

Comment:
@frankbu alot of the media has picked up on the word "central istiod". Would you mind adding a sentence that says something like "This was previously identified as `central Istiod`"?

Reply:
That belongs in Lin's blog, not here. We really should downplay that term IMO, because as we discussed on slack, it's not really in the center of anything.

Reply:
Ok I'll add it to my blog.